packages/devextreme/js/__internal/grids/grid_core/editing/m_editing_utils.ts

```diff
@@ -1,5 +1,9 @@
+import Guid from '@js/core/guid';
 import { isObject } from '@js/core/utils/type';
 
+const NEW_ROW_TEMP_KEY_PREFIX = '_DX_NEW_ROW_TEMP_KEY_';
```

Comment:
Not a comment, just thinking out loud

Maybe it will be better to make this prefix a little shorter.
E.g:
```suggestion
const NEW_ROW_TEMP_KEY_PREFIX = '_DX_TMP_';
```